### PR TITLE
Fixed issue #46 and 47.

### PR DESCRIPTION
**Pull request by HumanV2**
How does this look?

### CODE REVIEW

File: netbox_netisp/templates/netbox_netisp/customer.html [hunk 1]
```diff
@@ -59,9 +58,12 @@
                 </tr>
             </table>
         </div>
+
     </div>
 </div>
 
+<center><a class="btn btn-primary btn-sm"  href=" {% url 'plugins:netbox_netisp:account_add' customer_pk=object.pk %}">Create Account</a></center>
```
Comment (MajesticFalcon) [inline netbox_netisp/templates/netbox_netisp/customer.html]: Lets move this button to be with the other buttons please.

File: netbox_netisp/views.py [hunk 2]
```diff
@@ -110,7 +109,16 @@ class AccountListView(ObjectListView, View):
 class AccountEditView(ObjectEditView, View):
     queryset = Account.objects.all()
     model_form = forms.AccountForm
+    def get(self, request, *args, **kwargs):
+        print(kwargs)
```
Comment (MajesticFalcon) [inline netbox_netisp/views.py]: Remove debugging commands please

File: netbox_netisp/views.py [hunk 3]
```diff
@@ -110,7 +109,16 @@ class AccountListView(ObjectListView, View):
...
+        if "customer_pk" in kwargs:
+            account = Account()
+            customer = Customer.objects.get(pk=kwargs["customer_pk"]) #get current customer we're talking about
+            account.primary_applicant = customer
+            account.save() #saves to db
+            return redirect(account)
 
+        return super().get(request, *args, **kwargs)
```
Comment (MajesticFalcon) [inline netbox_netisp/views.py]: The logic flow here is not very clean. You are using an implicit else and I would prefer an explicit. Please refactor the code to be 'if customer pk then redirect else return super'

File: netbox_netisp/views.py [hunk 4]
```diff
@@ -110,7 +109,16 @@ class AccountListView(ObjectListView, View):
...
     queryset = Account.objects.all()
     model_form = forms.AccountForm
+    def get(self, request, *args, **kwargs):
+        print(kwargs)
+        if "customer_pk" in kwargs:
+            account = Account()
+            customer = Customer.objects.get(pk=kwargs["customer_pk"]) #get current customer we're talking about
+            account.primary_applicant = customer
+            account.save() #saves to db
+            return redirect(account)
```
Comment (MajesticFalcon) [inline netbox_netisp/views.py]: Get rid of comments

File: netbox_netisp/views.py [hunk 5]
```diff
@@ -284,3 +292,5 @@ class TicketView(ObjectView):
 
 class TicketDeleteView(ObjectDeleteView):
     queryset = Ticket.objects.all()
+    
+"""Register"""
```
Comment (MajesticFalcon) [inline netbox_netisp/views.py]: What is this? Delete this code if it isn't apart of this commit

File: netbox_netisp/templates/netbox_netisp/customer.html [hunk 6]
```diff
@@ -59,9 +58,12 @@
                 </tr>
             </table>
         </div>
+
```
Comment (MajesticFalcon) [inline netbox_netisp/templates/netbox_netisp/customer.html]: Remove this line of added whitespace

**Review by MajesticFalcon — CHANGES_REQUESTED** [review]
Looks good for your first commit, I have added some things for you to fix.

File: netbox_netisp/forms.py [hunk 7]
```diff
@@ -28,6 +28,7 @@ class Meta:
             "street_number",
             "street_name",
             "street_suffix",
+            "street_ordinance",
```
Comment (MajesticFalcon) [inline netbox_netisp/forms.py]: Logically, the street_direction should come before street name. By adjusting the position in the array, you will adjust the position it is displayed in the form.

**Review by MajesticFalcon — CHANGES_REQUESTED** [review]

**Review by HumanV2 — COMMENTED** [review]
Fixed location of street_ordinance as per format request.

**Comment by MajesticFalcon** [comment]
Fix issue #46 and #47

**Pull request merged**